nca/SchemeRunner.py

```diff
@@ -190,6 +190,11 @@ def run_queries(self, query_array):
             not_executed = 0
             self.check_fields_validity(query, 'query', allowed_elements)
             query_name = query['name']
+            if self.optimized_run == 'debug':
+                #optimization - currently only connectivityMap query has optimized implementation and can be compared
+                if not query.get('connectivityMap'):
+                    print(f'Skipping query {query_name} since it does not have optimized implementation yet')
+                    continue
```

Comment:
add a `TODO` comment to remove this after supporting additional queries.
Also, create an issue for supporting additional queries, and a link to this comment in the issue.

Reply:
Done